rcqt.so/include/metaPackageInfo.h

```diff
@@ -8,10 +8,15 @@ class PackageInfo {
 
 public:
   PackageInfo(const std::string& pkgname, const std::string& pkgmgr,
-      const std::string& cmd1, const std::string& cmd2 = "", const std::string& cmd3 = "") :
+      const std::string& cmd1, const std::string& cmd2, const std::string& cmd3) :
```

Comment:
same comment as above. can we take inputs as collection itself than cmd1, cmd2, cmd3? It is ambiguous to reader as what are cmd1 etc